index.html

```diff
@@ -1934,6 +1834,66 @@
       </section>
       <section>
         <h3>
+          Interface <a><code>PresentationReceiver</code></a>
+        </h3>
+        <pre class="idl">
+          interface PresentationReceiver {
+            readonly attribute Promise&lt;PresentationConnectionList&gt; connections;
+          };
+
+
+</pre>
+        <div class='issue'>
+          <p>
+            The text about "PresentationReceiver object" should probably move
+            to the section about the Presentation interface. This section is
+            about the interface, not its instance (the object).
+          </p>
+          <p>
+            Furthermore, we probably need a better way to explain how
+            PresentationConnectionList is initialized -- how the set of
+            presentations is populated.
+          </p>
+        </div>
+        <p>
+          The <a>PresentationReceiver</a> object is available to a <a>receiving
+          browsing context</a> in order to access the <a data-lt=
+          "controlling browsing context">controlling browsing context</a> and
+          communicate with it. The <a><code>PresentationReceiver</code></a>
+          object MUST be implemented in a <a>receiving browsing context</a>
+          provided by a <a>receiving user agent</a>.
+        </p>
+        <p>
+          The <a for="PresentationReceiver">connections</a> attribute MUST
+          return the result of running these steps:
+        </p>
+        <ol>
+          <li>Let <var>p</var> be a new <a>Promise</a>.
```

Comment:
Does this need to return a new Promise each time it's referenced in script?  Or should the algorithm return the same Promise each time?